local-cli/runAndroid/runAndroid.js

```diff
@@ -205,16 +218,18 @@ module.exports = {
   name: 'run-android',
   description: 'builds your app and starts it on a connected Android emulator or device',
   func: runAndroid,
-  options: [{
-    command: '--install-debug',
-  }, {
+  options: [ {
     command: '--root [string]',
     description: 'Override the root directory for the android build (which contains the android directory)',
     default: '',
   }, {
     command: '--flavor [string]',
     description: '--flavor has been deprecated. Use --variant instead',
   }, {
+    command: '--buildType [string]',
+    description: 'You can use `Release` or `Debug`. This creates a build based on the selected build type. If you want to use the `Release` build type make sure you have the `signingConfig` configurated at `app/build.gradle`. The default value is `Debug`.',
```

Comment:
I don't think you need to mention the default value here.

If our cli tools don't print that in the help command, we should fix that there.